Mods/vcmi/config/vcmi/english.json

```diff
@@ -113,6 +113,8 @@
 	"vcmi.battleOptions.movementHighlightOnHover.help": "{Movement Highlight on Hover}\n\nHighlight unit's movement range when you hover over it.",
 	"vcmi.battleOptions.rangeLimitHighlightOnHover.hover": "Show range limits for shooters",
 	"vcmi.battleOptions.rangeLimitHighlightOnHover.help": "{Show range limits for shooters on Hover}\n\nShow shooter's range limits when you hover over it.",
+	"vcmi.battleOptions.showStickyHeroInfoWindows.hover": "Permanently show heroes statistics windows",
```

Comment:
Perhaps drop "Permanently" from name?
Text already rather long, might be even longer with translations